### PR TITLE
adding admin-ui dependencies to integration-arquillian testsuite

### LINKED ISSUE #13465: Test Pipelines are unable to fetch Keycloak-admin-UI from Sonatype Repo
### Describe the bug

Keycloakk QE test pipelines use keycloak- version with current timestamp to differentiate between different builds but the current upstream repo fetches the keycloak-admin-ui from NIghhtly releases with a version 999-Snapshot which causes the failure.

### Version

19.0.0

### Expected behavior

Keycloak should be buit successfuly in the test pipelines

### Actual behavior

Following errors occurs in the test pipelines:
`[ERROR] Failed to execute goal on project integration-arquillian-servers-auth-server-undertow: Could not resolve dependencies for project org.keycloak.testsuite:integration-arquillian-servers-auth-server-undertow:jar:999-SNAPSHOT-pr-13393-20220728-124402: Could not find artifact org.keycloak:keycloak-admin-ui:jar:999-SNAPSHOT -> [Help 1]`

### How to Reproduce?

run the command :

mvn clean install -Pdistribution

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by nehachopra27**
Fix for issue : #13465

Current Test pipelines are failing due to missing keycloak-admin-ui dependancy in the testsuite.
https://test-pipelines.com/job/universal-test-pipeline-server/1967/

### CODE REVIEW

File: testsuite/integration-arquillian/pom.xml [hunk 1]
```diff
@@ -575,4 +593,19 @@
...
+            <url>https://s01.oss.sonatype.org/content/repositories/snapshots/</url>
+            <snapshots>
+                <enabled>true</enabled>
+                <updatePolicy>daily</updatePolicy>
+            </snapshots>
+            <releases>
+                <enabled>false</enabled>
+            </releases>
+        </repository>
+    </repositories>
```
Comment (stianst) [inline testsuite/integration-arquillian/pom.xml:609]: You shouldn't need to define the repo here as it's defined in the keycloak-dependencies-admin-ui-wrapper. Have you tested without it here?

Comment (nehachopra27) [reply testsuite/integration-arquillian/pom.xml:609]: I tried without it, and it says Could not find artifact org.keycloak:keycloak-admin-ui:jar:999-SNAPSHOT

Comment (stianst) [reply testsuite/integration-arquillian/pom.xml:609]: I'm okay with this for now as it's only introducing the Sonatype snapshot repo for the testsuite. We'll want to refactor something around this in the long run regardless, which me an @jonkoops can figure out somehow as it also relates to packaging of new account console + keycloak.js.

**Review by stianst — DISMISSED** [review]

File: testsuite/integration-arquillian/pom.xml (hunk 1 above)
Comment (jonkoops) [reply testsuite/integration-arquillian/pom.xml:609]: Perhaps we can log an issue for this?

**Review by miquelsi — DISMISSED** [review]

File: testsuite/integration-arquillian/pom.xml (hunk 1 above)
Comment (hmlnarik) [reply testsuite/integration-arquillian/pom.xml:609]: #13468

**Review by hmlnarik — APPROVED** [review]
Approving per before-squash approvals

**Pull request merged**